src/README.md

```diff
@@ -0,0 +1,82 @@
+# Hasura NDC Typescript (Deno) Connector
```

Comment:
How come there are two readmes?

Reply:
Yeah this is due to deno.land hosting using the /src subdirectory, so it needs to find a readme in there.

See: https://deno.land/x/hasura_typescript_connector@deno0.9

I'd like to just have one, but not sure how.